build/Makefile

```
@@ -20,15 +20,7 @@ pmm-ami-arm64:
 	cd docker/rpmbuild && docker build -t perconalab/rpmbuild:3 -f Dockerfile.el9 .
 
-rpmbuild-hetzner-el9: ## Build rpmbuild Docker image for Hetzner (Rocky Linux 9)
-	cd docker/rpmbuild && \
-	docker build \
-	-t perconalab/rpmbuild:3-hetzner \
-	-f Dockerfile.hetzner-el9 \
-	--build-arg GO_VERSION=$(GO_VERSION) \
-	.
-
-rpmbuild-all: rpmbuild-el9 rpmbuild-hetzner-el9 ## Build all rpmbuild Docker images
+rpmbuild-all: rpmbuild-el9 ## Build all rpmbuild Docker images
```

checkmake (0.3.2)

[warning] Target "rpmbuild-all" should be declared PHONY.

(phonydeclared)